ephios/core/notifications/backends.py

```diff
@@ -0,0 +1,84 @@
+from django.core.mail import EmailMultiAlternatives
+from django.utils.translation import gettext_lazy as _
+from webpush import send_user_notification
+
+from ephios.core.models.users import Notification
+from ephios.core.notifications.types import notification_type_from_slug
+from ephios.core.signals import register_notification_backends
+
+
+def installed_notification_backends():
+    for _, backends in register_notification_backends.send_to_all_plugins(None):
+        yield from (b() for b in backends)
+
+
+def enabled_notification_backends():
+    for _, backends in register_notification_backends.send(None):
+        yield from (b() for b in backends)
+
+
+class AbstractBackend:
```

Comment:
Can we do 
```suggestion
class AbstractNotificationBackend:
```

and also expand the name for subclasses?